ros2topic/ros2topic/verb/hz.py

```diff
@@ -316,7 +316,8 @@ def _rostopic_hz(node, topics, qos_args, window_size=DEFAULT_WINDOW_SIZE, filter
             msg_class,
             topic,
             functools.partial(rt.callback_hz, topic=topic),
-            qos_profile)
+            qos_profile,
+            raw=True if filter_expr is None else False)
```

Comment:
I should've mentioned this before Tomoya ran CI, but just a minor comment:

```suggestion
            raw=filter_expr is None)
```